.idea/gradle.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
On a clean clone of this branch, IntelliJ seems to import things properly and everything builds fine.

One odd thing is that IntelliJ doesn't seem to recognize the `androidMain` and `androidTest` directories as "modules" in the same way it does the other source set directories.  I'm not sure if this is something that could be configured to work differently, or if it's just how IntelliJ/Gradle understands the Android sources 🤔   I might try to compare the configuration to some other open source examples, but if you have any ideas I'm very open.  It's kind of driving me nuts 😉

Reply:
I can take a look, but nothing jumps out at me, off the top of my head.